packages/peregrine/lib/talons/CheckoutPage/__tests__/useCheckoutPage.spec.js

```diff
@@ -146,7 +147,8 @@ const getTalonProps = props => {
  * beforeAll
  */
 
-beforeAll(() => {
+beforeEach(() => {
+    jest.clearAllMocks();
```

Comment:
This shouldn't be necessary, since [we do this at the config level](https://github.com/magento/pwa-studio/blob/develop/jest.config.js#L245).